lib/src/main/java/com/orange/ods/compose/component/button/OdsIconButton.kt

```diff
@@ -22,6 +22,7 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.graphics.painter.Painter
 import androidx.compose.ui.graphics.vector.ImageVector
```

Comment:
Several imports including this one are unused and should be removed. Maybe Save Actions plugin is not working anymore ?